CHANGES.txt

```diff
@@ -28,6 +28,10 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Complete tests for Dictionary, env.keys() and env.values() for
       OverrideEnvironment. Enable env.setdefault() method, add tests.
 
+  From Joachim Kuebart:
+    - Suppress missing SConscript deprecation warning if `must_exist` is
```

Comment:
This is really only for when must_exist=False right? If so can you include that in the description here?

Reply:
true, when `must_exist=True` there is an error exit beforehand